package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "copy-text-to-clipboard",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
Don't bump version in a pull request. It's up to the maintainers when, how, and what version to bump.